packages/discovery-provider/src/utils/core.py

```diff
@@ -38,3 +38,11 @@ def is_indexing_core_em() -> Optional[bool]:
     if core_health:
         return core_health.get("indexing_entity_manager")
     return None
+
+
+# TODO: cache response to avoid excessive redis calls
+def is_indexing_core_plays() -> Optional[bool]:
+    core_health = get_core_health()
```

Comment:
looks like get_core_health is just the redis thing and json marshal. tbh - i think this is fine, no need to address this todo